### ISSUE TITLE
Event - Task - Project Task Shows Only End Date on Calendar

### ISSUE THREAD

**Issue by mariostevenson** [salesagility/SuiteCRM#254]
Hello,

RE: Event - Task - Project Task Shows Only End Date on Calendar

The task, event, project tasks are shown on the calendar as end dates. These items should have:
- start date and time
- description (should be mark-up compatible - reference TinyMCE)
- owner
- assigned
- group ownership
- group assigned
- invitees
- end date and time

Most important is a graphical line on the calendar stretching from item start date and time to item  end date and time. Also the resulting inclusions should show items equally matched as if the Project Management Gantt chart was used. It may be more efficient to make the Gantt chart Calendar global standard in SCRM instead of designing two (2) code fragments. 

I think the best way is using CSS for the SFX and then just adding the fields to show in the calendar. I think the problem is this calendar is too basic. JQuery has a JQuery calendar, but here's a topic on integration. The JQuery calendar integration would bring it all up to-date, but I seen a lot of calendars and it always come down to the same question: Does it integrate with Google Calendar? Would it be better to just integrate the Google Calendar API then integrate the Gantt chart, KPIs and Kanban?

-- MMS

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Event - Task - Project Task Shows Only End Date on Calendar](https://trello.com/c/YNyG2EVU/153-event-task-project-task-shows-only-end-date-on-calendar)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
